src/ott-analytics.js

```diff
@@ -156,6 +165,7 @@ export default class OttAnalytics extends BasePlugin {
    */
   _onEnded(): void {
     this._isPlaying = false;
+    this._isFinished = true;
```

Comment:
should reset to `false` on `REPLAY`

Reply:
replay is not handled at all at the moment so deferring the fix to another PR